streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java

```diff
@@ -1,15 +1,11 @@
 package org.infinispan.tutorial.simple.streams;
 
-import java.io.Serializable;
-import java.util.function.BinaryOperator;
-import java.util.function.Function;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
-
 import org.infinispan.Cache;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
 import org.infinispan.manager.DefaultCacheManager;
-import org.infinispan.stream.CacheCollectors;
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
```

Comment:
The Infinispan code style is to import `javax` and `java` packages first, let's keep it the same in the tutorials.

Reply:
I don't understand what happened here with formatters